config/Config-build.in

```diff
@@ -7,6 +7,13 @@
 
 menu "Global build settings"
 
+	config JSON_CREATE_IMAGE_INFO
```

Comment:
Below you are using JSON_ADD_IMAGE_INFO ...

Reply:
Fixed, thanks!